packages/views.py

```diff
@@ -83,53 +120,63 @@ def partner_new_order(request):
         origin_area = request.POST.get('origin_area')
         origin_street = request.POST.get('origin_street')
         origin_street_number = request.POST.get('origin_street_number')
-        origin_address = get_address(origin_city, origin_area, origin_street, origin_street_number)
+        origin_address = get_or_create_address(origin_city, origin_area, origin_street, origin_street_number)
 
         destination_city = request.POST.get('destination_city')
         destination_area = request.POST.get('destination_area')
         destination_street = request.POST.get('destination_street')
         destination_street_number = request.POST.get('destination_street_number')
-        destination_address = get_address(destination_city, destination_area, destination_street, destination_street_number)
+        destination_address = get_or_create_address(destination_city, destination_area, destination_street, destination_street_number)
 
-        rate = float(request.POST.get('rate'))
+        rate = float(request.POST.get('rate').replace('₪', ''))
         phone_number = request.POST.get('phone_number')
+        full_name = request.POST.get('full_name')
 
-        Package.objects.create(origin=origin_address, destination=destination_address, order=order, rate=rate, phone_number=phone_number)
+        Package.objects.create(origin=origin_address, destination=destination_address, order=order, rate=rate,

```

Comment:
Why do you create a new one each time?

Reply:
I create a new Package each time that there is a new package... What do you mean?